src/Dispatcher.ts

```diff
@@ -43,8 +44,9 @@ export class Dispatcher {
         });
 
         ActionMoveCursor.updatePreferedColumn();
-
-        this.switchMode(ModeID.NORMAL);
+        // set to default mode of Configuration
+        console.log(Configuration.defaultModeID);
```

Comment:
Please remove debug log since this will drawdown the performance.

Besides, comment is not needed when the code can explain itself.